src/ssl.c

```diff
@@ -12751,6 +12778,12 @@ int wolfSSL_Cleanup(void)
     #endif
 #endif
 
+#ifdef WOLFSSL_CHECK_MEM_ZERO
```

Comment:
Would this be better in `wolfCrypt_Cleanup()`?

Reply:
Yep! Fixed.